docs/operator-guides/refreshes.md

```diff
@@ -0,0 +1,128 @@
+
+When performing a Refresh Sync, any records you’ve already imported will be imported again for the stream(s) being refreshed (assuming they are still present within the source), leading to a slower and more expensive sync than you might be used to. Refresh syncs will block any scheduled syncs from occurring until they are complete. During a Refresh Sync, the streams that are not being refreshed will also sync normally in order to lower data latency . All refresh syncs are resumable, if the underlying source and destination support [checkpointing](/understanding-airbyte/airbyte-protocol/#state--checkpointing).
+
+Depending on the sync mode you’ve selected, you’ll be able to choose between the following refresh options:
+
+| Stream Sync Mode               | Refresh stream and remove records | Refresh stream and retain records | Clear     |
+| ------------------------------ | --------------------------------- | --------------------------------- | --------- |
+| Incremental \| Append + Dedupe | Available                         | Available                         | Available |
+| Full Refresh \| Overwrite      | Not available                     | Not available                     | Available |
+| Full Refresh \| Append         | Not available                     | Not available                     | Available |
+| Incremental \| Append          | Not available                     | Available                         | Available |
+
+For supported destinations, and depending on the sync mode you have selected, there are two types of refreshes - "retaining history" and "removing history". Both kinds of Refresh Syncs can be triggered from the UI and via the Airbyte API at either the stream or connection levels (although both might not be available depending on your sync mode).
+
+![Are you sure you want to refresh data](../.gitbook/assets/refreshes/are-you-sure-refresh.png)
+
+## Refresh and Remove Records
+
+Refresh and Remove Records Syncs will only display data in the destination’s final tables that was newly synced since the refresh was initiated. This is the simplest type of refresh, and leads to table mirroring, if you are using an overwrite sync mode.
+
+:::warning
+
+Not all sources keep their history forever. If you perform a Refresh and Remove Records Sync, and your source does not retain all of its records, this may lead to data loss.
+
+:::
+
+## Refresh and Retain Records
+
+Refresh and Retain Records Syncs keep the records you had previously, and merge in the new data from this sync onward with the data previously stored in your destination. Using a Refresh and Retain Records Sync with a Deduplicated Sync mode is the best way to recover from an unreliable source.
```

Comment:
Have we defined what an `unreliable source` is anywhere? If not, we should, and link to it here. I'm not sure it will be a widely understood term.

Reply:
I don't think so. I added `(e.g. lossy or buggy)`